source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Build/002_PSP_PIMS_Build.sql

```diff
@@ -15530,33 +15553,27 @@ SELECT PROP.PROPERTY_ID
                          AND TAKE.TAKE_STATUS_TYPE_CODE  = N'COMPLETE') THEN CONVERT([bit],(1))
          WHEN EXISTS (SELECT 1
                       FROM   PIMS_TAKE                      TAKE JOIN
-                             PIMS_PROPERTY_ACQUISITION_FILE PRAF   ON PRAF.PROPERTY_ACQUISITION_FILE_ID = TAKE.PROPERTY_ACQUISITION_FILE_ID
+                             PIMS_PROPERTY_ACQUISITION_FILE PRAF   ON PRAF.PROPERTY_ACQUISITION_FILE_ID = TAKE.PROPERTY_ACQUISITION_FILE_ID    
                                                                   AND PRAF.PROPERTY_ID                  = PROP.PROPERTY_ID
                       WHERE  TAKE.IS_NEW_LICENSE_TO_CONSTRUCT = 1
                          AND TAKE.TAKE_STATUS_TYPE_CODE       = N'COMPLETE') THEN CONVERT([bit],(1))
-         WHEN EXISTS (SELECT 1
-                      FROM   PIMS_TAKE                      TAKE                                                                            JOIN
-                             PIMS_PROPERTY_ACQUISITION_FILE PRAF   ON PRAF.PROPERTY_ACQUISITION_FILE_ID = TAKE.PROPERTY_ACQUISITION_FILE_ID
-                                                                  AND PRAF.PROPERTY_ID                  = PROP.PROPERTY_ID                  JOIN
-                             PIMS_PROPERTY_LEASE            PRLS   ON PRLS.PROPERTY_ID                  = PROP.PROPERTY_ID                  JOIN
-                             PIMS_LEASE                     LEAS   ON LEAS.LEASE_ID                     = PRLS.LEASE_ID
-                      WHERE  LEAS.LEASE_PAY_RVBL_TYPE_CODE IN (N'PYBLMOTI', N'PYBLBCTFA')
-                         AND TAKE.TAKE_STATUS_TYPE_CODE     = N'COMPLETE') THEN CONVERT([bit],(1))
+         WHEN EXISTS (SELECT 1                                                                                                                                    
+                      FROM   PIMS_TAKE
+                      WHERE  IS_ACTIVE_LEASE       = 1
+                         AND TAKE_STATUS_TYPE_CODE = N'COMPLETE') THEN CONVERT([bit],(1))
          ELSE CONVERT([bit],(0))
-         END AS IS_OTHER_INTEREST 
-     , IIF(EXISTS (SELECT 1
+         END AS IS_OTHER_INTEREST                                                                 
+     , IIF(EXISTS (SELECT 1                        
                    FROM   PIMS_PROPERTY_ACQUISITION_FILE PRAF JOIN
                           PIMS_ACQUISITION_FILE          ACQF   ON ACQF.ACQUISITION_FILE_ID = PRAF.ACQUISITION_FILE_ID
                                                                AND PRAF.PROPERTY_ID         = PROP.PROPERTY_ID
-                   WHERE  ACQF.COMPLETION_DATE                   IS NOT NULL
-                      AND ACQF.ACQUISITION_FILE_STATUS_TYPE_CODE <> N'CANCEL'), CONVERT([bit],(1)), CONVERT([bit],(0))) AS HAS_ACTIVE_ACQUISITION_FILE
+                   WHERE  ACQF.ACQUISITION_FILE_STATUS_TYPE_CODE IN (N'DRAFT', N'COMPLT')), CONVERT([bit],(1)), CONVERT([bit],(0))) AS HAS_ACTIVE_ACQUISITION_FILE
```

Comment:
Should this be active or draft instead of completed or draft?

Reply:
active or draft